conf/messages.en

```diff
@@ -1056,6 +1056,12 @@ declaration.summary.references.ducr.change = DUCR
 declaration.summary.references.lrn = LRN
 declaration.summary.references.lrn.change = LRN
 declaration.summary.parties = Parties
+declaration.summary.parties.eidr = Is this an EIDR?
+declaration.summary.parties.eidr.change = Is this an EIDR?
```

Comment:
I forgot about this. I was wondering during my ticket why we have two message keys for the same field.

It's not something to do with this PR, this is the pattern that we're using right now, but do you know if there is any reason of this?

I think that in the future we should remove those `change` messages and use everywhere one message.

Reply:
ah, I see this, so the second message should start with lower case.

We have a few messages incorrect. I'm sure about my ticket.

Reply:
I have no idea, looked how it is implemented so far. Don't see much value in having separate messages for that.